libs/template/renderer.go

```diff
@@ -313,8 +319,7 @@ func (r *renderer) persistToDisk() error {
 		_, err := os.Stat(path)
 		if err == nil {
 			return fmt.Errorf("failed to initialize template, one or more files already exist: %s", path)
-		}
-		if err != nil && !errors.Is(err, fs.ErrNotExist) {
+		} else if !errors.Is(err, fs.ErrNotExist) {
```

Comment:
Nit: isn't this easier to read without the `else if`?

Reply:
yes, it is